### DISCUSSION TITLE
mypy recognizes Config.config_file_name as `Union[str, bytes, PathLike[str], PathLike[bytes], int]`

### DISCUSSION THREAD

**Discussion by georgegarrington** [sqlalchemy/alembic#1077] [Usage Questions]
I came across this issue when running mypy on the alembic/env.py file, using mypy 0.971 and alembic 1.8.1. The docs say Config.config_file_name is `Optional[str]`: https://alembic.sqlalchemy.org/en/latest/api/config.html#:~:text=Filesystem%20path%20to%20the%20.ini%20file%20in%20use. It looks to me this could be from the standard library mypy stubs updating the type of filepath to `Union[str, bytes, PathLike[str], PathLike[bytes], int]`

**Comment by CaselIT** [comment]
Hi,

this seems to be for alembic.
edit: moved
